docs/draft-dontula-lsr-yang-dynamic-flooding-02.xml

```diff
@@ -1353,15 +1375,15 @@ module ietf-isis-dynflood {
      </t>
      <figure align="left">
        <artwork align="left"><![CDATA[
-        name: ietf-ospf-df
-        namespace: urn:ietf:params:xml:ns:yang:ietf-dynamic-flooding-ospf
-        prefix: ospf-df
-        reference: RFC XXXX
-
-        name: ietf-isis-dynflood
-        namespace: urn:ietf:params:xml:ns:yang:ietf-isis-dynflood
-        prefix: isis-dynflood
-        reference: RFC XXXX
+       name: ietf-ospf-df
```

Comment:
How about ietf-ospf-dynflood, for symmetry?

Reply:
So your recommendation is that
1. name changes from ietf-ospf-df to ietf-ospf-dynflood
2. prefix changes from ospf-df to ospf-dynflood
I'll do that.

Should the namespace also change from
namespace "urn:ietf:params:xml:ns:yang:ietf-dynamic-flooding-osp";
to
namespace "urn:ietf:params:xml:ns:yang:ietf-ospf-dynflood";

Changing namespace will also need a change to module name and file name.

Reply:
The namespace is fine.
I'm suggesting changing the name and the prefix.